docs/run/nodes/nodes.md

```diff
@@ -254,7 +252,7 @@ Ceramic state store AWS S3 policy for the access key
 
 ---
 
-The 3Box Labs team runs Ceramic and out-of-process IPFS nodes in AWS ECS with 4 vCPU and 8 GB of memory allocated for each node.
+The 3Box Labs team runs Ceramic and remote IPFS nodes in AWS ECS with 4 vCPU and 8 GB of memory allocated for each node.
```

Comment:
I vote to remove this section. We don't yet know how much go-ipfs will use and this is not the actual recommendation even for js-ipfs.

Reply:
done